src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -246,7 +246,8 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
       // Re-init interactive, this time using a new mode (report or runtime).
       phoneRef.current = new iframePhone.ParentEndpoint(iframeRef.current, initInteractive);
       setSendCustomMessage((message: ICustomMessage) => {
-        phoneRef.current?.post(message.type, message.content);
+        // TODO: structure message type and content correctly
+        phoneRef.current?.post("customMessage", message);
```

Comment:
My understanding was that we post "customMessage" and the message itself contains a `type` and optional `content`.  In the TE case that we've been working on, the `type` on the `message` would specify showing/hiding overlays and the `content` could be empty.  Do I have this correct or am I not understanding the message structure?

Reply:
Yes that is how I thought it should work.

Reply:
Thanks.  I removed the now obsolete comment as well.  I would vote that we merge this so I can move on to other work.